src/app/googDevice/client/DeviceTracker.ts

```diff
@@ -421,6 +421,77 @@ export class DeviceTracker extends BaseDeviceTracker<GoogDeviceDescriptor, never
         this.ShellClient = ShellClient.start(shellParams, this.params);
     }
 
+    private buildGPSMockingTab(): void {
+        const divHtml = html`
+            <div>
+                <label for="lat-input">Latitude:</label>
+                <input type="text" id="lat-input" placeholder="Enter latitude" />
+
+                <label for="lng-input">Longitude:</label>
+                <input type="text" id="lng-input" placeholder="Enter longitude" />
+
+                <button id="mock-btn" class="button-primary">Mock</button>
+            </div>
+            <hr class="full-line" />
+            <div id="map" style="width: 100%; height: 400px;"></div>
```

Comment:
Is it possible to use relative size? `400px` could look very different on different screens?